src/common/bankarskiSistem/BankarskiSistem.java

```diff
@@ -1,41 +1,252 @@
+                        System.out.println();
+                    }
+                    case 8 -> {
+                        zatvoriRacun(banka, sc);
+                        System.out.println();
+                    }
+                    case 9 -> System.exit(1);
+                }
+
+            }
+        }
 
-            Banka bankaIntesa = new Banka("Intesa", "Adresa 1", new Kurs(kursnaLista1));
-            Banka bankaErste = new Banka("Erste", "Adresa 2", new Kurs(kursnaLista2));
+    private static Banka odabirBanke(ArrayList<Banka> banke, Scanner sc) {
+        for (int i = 0; i < banke.size(); i++) {
+            System.out.println(i+1 + ")");
+            System.out.println(banke.get(i));
+            System.out.println();
+        }
+        System.out.println("Odaberi redni broj banke: ");
 
-            Korisnik korisnik = new Korisnik("Pera", "Peric", "Topolska 18", "2001999710033");
-            Korisnik korisnik2 = new Korisnik("Marko", "Markovic", "Topolska 19", "1001989710043");
+        int odabir = -1;
+        if (sc.hasNextLine()) {
+            odabir = Integer.parseInt(sc.nextLine());
+        }
 
-            Racun racun = new Racun(Valuta.EUR, 123, korisnik, bankaIntesa);
-            Racun racun2 = new Racun(Valuta.RSD, 124, korisnik, bankaIntesa);
-            Racun racun3 = new Racun(Valuta.USD, 100, korisnik, bankaErste);
+        if (odabir <= 0 || odabir > banke.size())
+            throw new IndexOutOfBoundsException("Nevalidan broj banke");
 
-            bankaIntesa.dodajRacun(racun, korisnik);
-            bankaIntesa.dodajRacun(racun2, korisnik);
-            bankaErste.dodajRacun(racun3, korisnik2);
-            korisnik.uplata(racun, 1000); //1000 eur
-            korisnik.uplata(racun2, 11700);  //din - 99e
+        return banke.get(odabir-1);
+    }
 
-            System.out.println(korisnik.stanjeSvihRacuna(Valuta.EUR));
+    private static int uslugeBanke(Scanner sc) {
+        System.out.println("1 - Postani Korisnik:\n" +
+                "2 - Uplata\n" +
+                "3 - Isplata\n" +
+                "4 - Otvor novi racun\n" +
+                "5 - Stanje svih racuna korisnika\n" +
+                "6 - Stanje\n" +
+                "7 - Transfer\n" +
+                "8 - Zatvori racun\n" +
+                "9 - Izlaz");
+        return Integer.parseInt(sc.nextLine());
+    }
+
+    private static Korisnik kreirajKorisnika(Banka banka,Scanner sc) {
+
+        System.out.println("Unesite ime, prezime, adresu i jmbg: ");
+
+        String ime = sc.nextLine();
+        String prezime = sc.nextLine();
+        String adresa = sc.nextLine();
+        String jmbg = sc.nextLine();
+
+        if(jmbg.length() != 13 || !jmbg.matches("[0-9]+")) {
+            System.out.println("Nevalidan jmbg!");
+            return null;
+        }
+
+        Korisnik korisnik = new Korisnik(ime, prezime, adresa, jmbg);
+        banka.getKorisnici().add(korisnik);
```

Comment:
moze li ova lista biti null ili ste se pobrinuli za to na drugom mestu?

Reply:
Ne moze biti null jer se u konstruktoru uvek inicijalizuje lista koja je prazna.

Reply:
super